redi-connect-front/src/components/organisms/Navbar.tsx

```diff
@@ -9,9 +9,21 @@ import { useHistory } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
```

Comment:
Can we change main component name in this document to Navbar as the filename. I know you didn't have anything to do with it but it is still incorrect ATM because ti grew in another direction.

Reply:
`LoggedOutNavbar` renamed to `Navbar`